CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [1.0.1] = 2020-09-22
```

Comment:
We already have 1.0.1 but unfortunately we did not keep the change log for that hotfix.
Per CONTRIBUTION_GUIDELINES.md:
Update the "Unreleased" section in the CHANGELOG.md with the change made in this PR.

Reply:
I moved the changes to "Unreleased" under the headers "Added" and "Changed".